src/App/IntroPageAdmin/IntroPageAdmin.java

```diff
@@ -238,7 +240,30 @@ public void initialize(URL location, ResourceBundle resources) {
         getClientTableData();
     }
 
-    public void ProfileBtnAction(ActionEvent event) {
+    public JFXButton profileBtn;
+
+    public void ProfileBtnAction(ActionEvent event) throws SQLException {
+        if(event.getSource() == profileBtn){
+            FXMLLoader Loader = new FXMLLoader();
+
+            Loader.setLocation(getClass().getResource("../AdminProfile/AdminProfile.fxml"));
+
+            try {
+                Loader.load();
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+
+            AdminProfile adminProfile = Loader.getController();
+            adminProfile.setAdminId(getAdminId());
```

Comment:
You need to add `adminProfile.setUserRole(getUserRole());`
Otherwise when someone clicks home button and go to Intropage `userRole` become `null`